project/Build.scala

```diff
@@ -293,6 +316,26 @@ object StorehausBuild extends Build {
     javaOptions in run <++= (fullClasspath in Runtime) map { cp => Seq("-cp", sbt.Build.data(cp).mkString(":")) }
   ).dependsOn(storehausCore, storehausAlgebra, storehausCache)
 
+  def cassandraDeps(scalaVersion: String) = if (!isScalaGreater29x(scalaVersion)) Seq() else Seq(
+    "com.twitter" %% "algebird-core" % algebirdVersion,
+    "com.google.code.findbugs" % "jsr305" % "1.3.+",
+    "com.twitter" %% "bijection-core" % bijectionVersion,
+    "com.datastax.cassandra" % "cassandra-driver-core" % cassandraDriverVersion classifier "shaded" exclude("io.netty", "*"),
+    "com.websudos" %% "phantom-dsl" % "1.5.0" exclude ("com.datastax.cassandra", "cassandra-driver-core"),
+    withCross("com.twitter" %% "util-zk" % utilVersion)  exclude ("com.google.guava", "guava"),
+    bogus210Version("com.chuusai" %% "shapeless" % "2.0.0"),
```

Comment:
2.2.5 doesn't seem to distinguish between scala 2.10.x any more. is it safe to assume that it will be working with all 2.10s?